cmd/sign_test.go

```diff
@@ -28,8 +32,31 @@ var _ = Describe("SignCmd", func() {
 	)
 
 	It("with valid args", func() {
+		old := os.Stdout // keep backup of the real stdout
+		r, w, _ := os.Pipe()
+		os.Stdout = w
+
 		err := runSign(config, []string{objectID, "get", "15m"})
+
+		outC := make(chan string)
+		// copy the output in a separate goroutine so printing can't block indefinitely
+		go func() {
+			var buf bytes.Buffer
+			io.Copy(&buf, r)
```

Comment:
Well, that's some boilerplate code, in a test case, for a possibly unused feature. So, I would not invest much time into.

I'm also unsure what to do in case of a copy error, I admit. But feel free to push a patch implementing your idea if you're up to it 👍